src/message-manager/TypedMessageManager.ts

```diff
@@ -78,11 +78,6 @@ export class TypedMessageManager extends AbstractMessageManager<
    */
   name = 'TypedMessageManager';
 
-  /**
-   * List of required sibling controllers this controller needs to function
-   */
-  requiredControllers = ['NetworkController'];
```

Comment:
This controller doesn't use the network controller. This should be removed in a separate PR.

Reply:
Removed in #416